exercises/all-your-base/canonical-data.json

```diff
@@ -153,7 +153,7 @@
         "digits": [0],
         "outputBase": 10
       },
-      "expected": {"error": "input base must be >= 2"}
+      "expected": { "error": "input base must be >= 2" }
```

Comment:
Seems like prettier is a lot less opinionated about JSON than jq. That's good for arrays, but bad for cases like this where it would be nice to have fields (`error` in this case) on their own lines.

Does prettier have any options to force this ?

Reply:
I don't think so. Prettier has (purposefully) few options: https://prettier.io/docs/en/options.html